cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -398,17 +398,14 @@ def send_tokens(kube_control):
 
 @when_not('kube-control.connected')
 def missing_kube_control():
-    """Inform the operator they need to add the kube-control relation.
+    """Inform the operator master is waiting for a relation to workers.
 
     If deploying via bundle this won't happen, but if operator is upgrading a
     a charm in a deployment that pre-dates the kube-control relation, it'll be
     missing.
 
     """
-    hookenv.status_set(
-        'blocked',
-        'Relate {}:kube-control kubernetes-worker:kube-control'.format(
-            hookenv.service_name()))
+    hookenv.status_set('blocked', 'Waiting for workers.')
```

Comment:
Should this be `waiting` instead of `blocked`?

I _think_ the intention of https://github.com/juju-solutions/bundle-canonical-kubernetes/issues/309#issuecomment-314089288 is to stop indicating that we're blocked when we're not. @tvansteenburgh can you weigh in here?